components/StyledInputAmount.js

```diff
@@ -98,7 +99,6 @@ const CurrencyPicker = ({ availableCurrencies, value, onChange }) => {
       onInputChange={inputValue => (inputValue.length <= 3 ? inputValue : inputValue.substr(0, 3))} // Limit search length to 3 characters
       styles={{
         control: {
-          border: 'none',
```

Comment:
This creates a styling regression where we have duplicate borders

![image](https://user-images.githubusercontent.com/1556356/157839703-04564f5e-d208-4854-8ca0-b5e42562428c.png)

Reply:
Ops, makes sense. I've added the styles conditionally to handle this. 👍🏽